src/Novu/Interfaces/ITopicClient.cs

```diff
@@ -7,60 +8,161 @@ public interface ITopicClient
     /// <summary>
     /// Create a topic
     /// </summary>
-    /// <param name="dto"></param>
-    /// <returns></returns>
-    public Task CreateTopicAsync(TopicCreateDto dto);
+    /// <example>
+    /// <para>
+    /// <code>
+    /// var topicRequest = new TopicCreateDto
+    /// {
+    ///     Key = $"test:topic:{Guid.NewGuid().ToString()}",
+    ///     Name = "Test Topic",
+    /// };
+    ///
+    /// var topic = await client.Topic.CreateTopicAsync(topicRequest);
+    /// </code>
+    /// </para>
+    /// </example> 
+    /// <param name="dto">
+    /// <see cref="TopicCreateDto"/>
+    /// </param>
+    /// <returns>
+    /// <see cref="TopicCreateResponseDto"/>
+    /// </returns>
```

Comment:
Any particular reason why we have the docblocks like this in different html tags?

Reply:
We don't have them in the other code, so I was wondering why we need them all of a sudden now.